src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -309,10 +309,10 @@ export default class AddStaffNotationDialog extends Mixins(DateMixin) {
       this.enableValidation = false
 
       // reset notation form
-      this.$refs.notationFormRef.reset()
+      this.$refs.notationFormRef?.reset()
```

Comment:
SonarCloud recommended using conditional chaining here instead of my previous syntax.

(Previous codewas `this.$refs.notationFormRef&& this.$refs.notationFormRef.reset()`)